CONTRIBUTING.md

```diff
@@ -72,3 +72,13 @@ OR
 * The module is under the Node.js foundation Github org
 OR
 * The module is identified as an important module by a Node.js Working Group
+
+#### Procedure
+
+After making sure you adhere to the above requirements, do the
+following:
+
+1. Make sure the `citgm <module>` tests pass (`./bin/citgm.js`)
+1. Add the module to
```

Comment:
tbqh I can't recall... but it may default to using npm to test the module rather than grabbing the tarball from github (what it does if it is in the lookup). We need the prefix information in order to ensure we grab the tarball